File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
  let answer = [];
  const stringToArray = s.split('');
  for (const string of stringToArray) {
    const target = stringToArray.filter((targetNumber) => targetNumber === string)
    if (target.length === 1) {
      answer.push(string)
    }
  }
  return answer.sort().join('');
}

File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    obj = {}
    answer = []
    
    for i in range(len(s)):
        if s[i] not in obj:
            obj[s[i]] = 1
        else:
            obj[s[i]] += 1
            
    for str in obj:
        if obj[str] == 1:
            answer.append(str)         
    
    answer = sorted(answer)
    return "".join(answer)